src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -383,12 +383,12 @@ private static object SanitizeSecurityArtifact(object arg)
                 return "null";
 
             if (IdentityModelEventSource.LogCompleteSecurityArtifact && arg is ISafeLogSecurityArtifact)
-                return (arg as ISafeLogSecurityArtifact).UnsafeToString();
+                return ((ISafeLogSecurityArtifact)arg).UnsafeToString();
 
             return arg;
         }
 
-        private static string RemovePII(object arg)
+        private static string? RemovePII(object arg)
```

Comment:
this doesn't look like it can return null?

Reply:
Correct...why was this added as a possible null return?

Reply:
It used in line 375 as a return value

 if (!IdentityModelEventSource.ShowPII)
                return string.Format(CultureInfo.InvariantCulture, format, args.Select(RemovePII).ToArray()); 

IDE throws error code CS8603-Possible null reference return I can also suppress it?

Reply:
should be `return arg.ToString()!; // the null check is done in FormatInvariant`